client/me/privacy/main.jsx

```diff
@@ -53,6 +57,23 @@ const Privacy = createReactClass( {
 		);
 	},
 
+	componentDidUpdate( oldProps ) {
+		const { dpaRequest, translate } = this.props;
+		const { dpaRequest: oldDpaRequest } = oldProps;
+
+		if ( dpaRequest.status === 'success' && dpaRequest.status !== oldDpaRequest.status ) {
+			this.props.successNotice(
+				translate( 'Request successful! We are sending you our DPA via email' )
```

Comment:
Could we please add a translator comment to explain `DPA` and all the other occurrences where DPA is just used as an acronym? Thanks!

Reply:
Yes, thanks for mentioning it, @akirk. 🙇

Reply:
```suggestion
				translate( 'Request successful! We are sending you our DPA via email', { comment: 'A Data Processing Addendum (DPA) is a document to assure customers, vendors, and partners that their data handling complies with the law.' } )
```